newsfragments/1312.misc

```diff
@@ -0,0 +1 @@
+Use xia2 reporting style for dials.scale and dials.merge merging statistics summary
```

Comment:
Prefer "add Table 1, following the style of output from `xia2`" or something (i.e. `xia2` does not own the style